service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderController.java

```diff
@@ -90,7 +90,13 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
             ValidateFamilyManCaseNumberGroup.class);
 
         if (errors.isEmpty()) {
-            childrenService.addPageShowToCaseDetails(caseDetails, caseData.getAllChildren());
+            int numOfChildren = caseData.getAllChildren().size();
+            caseDetails.getData().put("pageShow", numOfChildren <= 1 ? "No" : "Yes");
+
+            if (numOfChildren <= 1) {
+                caseDetails.getData().put("orderAppliesToAllChildren", "Yes");
```

Comment:
It would have to be `YesNo.Yes.getValue()`, unless something has changed recently. I think I remember changing the JsonProperty of the enum it broke something else. Although this was a couple of months ago so I could be misremembering.